tests/run_test.sh

```diff
@@ -175,10 +175,11 @@ compute_petbounds
 atparse < ${PATHRT}/parm/${NEMS_CONFIGURE:-nems.configure} > nems.configure
 
 # remove after all tests pass
```

Comment:
Should we change this comment to say that the TASKS variable is forced to the total number of tasks determined by the script. And then also remove the commented out test block?

Reply:
@junwang-noaa I see the last change you made here. Looks good to me now.